lib/core/versions/latest/PatchAction.ts

```diff
@@ -0,0 +1,7 @@
+export enum PatchAction {
```

Comment:
Consider making default export enum for style consistency (see `OperationType.ts`)